release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -0,0 +1,810 @@
+application development and cluster administrator personas.
+
+See
+xref:../install/configuring_inventory_file.adoc#install-config-configuring-inventory-file[Configuring
+Your Inventory File] for details on configuring the admin console.
+
+[[ocp-311-security]]
+=== Security
+
+[[ocp-311-control-sharing-pid-namespace-between-containers]]
+==== Control Sharing the PID Namespace Between Containers (Technology Preview)
+
+Control Sharing the PID Namespace Between Containers is currently in
+xref:ocp-311-technology-preview[Technology Preview] and not for production
+workloads.
+
+Use this feature to configure cooperating containers in a pod, such as a log
+handler sidecar container, or to troubleshoot container images that do not
+include debugging utilities like a shell.
+
+* The feature gate `PodShareProcessNamespace` is set to `false` by default.
+* Set `feature-gates=PodShareProcessNamespace=true` in  the API server,
+controllers, and kubelet.
+* Restart the API server, controller, and node service.
+* Create a pod with the specification of `shareProcessNamespace: true`.
+* Run `oc create -f <pod spec file>`.
+
+*Caveats*
+
+When the PID namespace is shared between containers:
+
+* Sidecar containers are not isolated.
+* Environment variables are now visible to all other processes.
+* Any *kill all* semantics used within the process are now broken.
+* Any `exec` processes from other containers will now show up.
+
+See
+xref:../dev_guide/expanding_persistent_volumes.adoc#expanding_persistent_volumes[Expanding
+Persistent Volumes] for more information.
+
+[[ocp-311-github-enterprise-added-as-auth-Provider]]
+====  GitHub Enterprise Added as Auth Provider
+
+GitHub Enterprise is now added as an auth provider. OAuth facilitates a token
+exchange flow between {product-title} and GitHub or GitHub Enterprise. You can
+use the GitHub integration to connect to either GitHub or GitHub Enterprise. For
+GitHub Enterprise integrations, you must provide the `hostname` of your instance
+and can optionally provide a `ca` certificate bundle to use in requests to the
+server.
+
+See xref:../install_config/configuring_authentication.adoc#GitHub[Configuring
+Authentication and User Agent] for more information.
+
+[[ocp-311-sspi-connection-support-on-windows]]
+==== SSPI Connection Support on Microsoft Windows (Technology Preview)
+
+SSPI Connection Support on Microsoft Windows is currently in
+xref:ocp-311-technology-preview[Technology Preview] and not for production
+workloads.
+
+`oc` now supports the Security Support Provider Interface (SSPI) to allow for SSO
+flows on Windows. If you use the request header identity provider with a
+GSSAPI-enabled proxy to connect an Active Directory server to {product-title},
+users can automatically authenticate to {product-title} by using the `oc`  command
+line interface from a domain-joined Windows computer.
+
+See xref:../install_config/configuring_authentication.adoc#GitHub[Configuring
```

Comment:
Note to self: update this to be windows-sspi-using-request-header when the docs merge